spec/graphql/resolvers/subscriptions_resolver_spec.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Resolvers::SubscriptionsResolver, type: :graphql do
+  let(:query) do
+    <<~GQL
+      query {
+        subscriptions(limit: 5) {
+          collection { id externalId plan { code } }
+          metadata { currentPage, totalCount }
+        }
+      }
+    GQL
+  end
+
+  let(:membership) { create(:membership) }
+  let(:plan) { create(:plan, organization:) }
+  let(:organization) { membership.organization }
+  let(:customer) { create(:customer, organization:) }
+
+  before do
+    plan
+    customer
+  end
+
+  it 'returns a list of subscriptions' do
```

Comment:
Maybe add a test with two subscriptions attached to a different plan? So we ensure only one subscription is returned :)